<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#131239</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>740</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Georgia&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Security answer:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="f_getpass">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>550</y>
     <width>250</width>
     <height>65</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
padding:0px 5px 0px 5px;
background-color:green;
color:white;</string>
   </property>
   <property name="text">
    <string>Get Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Copperplate Gothic Light&quot;;
color:white;
</string>
   </property>
   <property name="text">
    <string>FORGOT PASSWORD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_answer">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px 10px 0px 10px;
border-radius:5px;
border:3px solid lightgreen;
background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Your Answer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f_gotologin">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>660</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
padding:0px 5px 0px 5px;
background-color:green;
color:white;</string>
   </property>
   <property name="text">
    <string>Go to Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_responseerror">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Meiryo UI&quot;;
color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Georgia&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Enter Your Email ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_emailid">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px 10px 0px 10px;
border-radius:5px;
border:3px solid lightgreen;
background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Your Email ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_responseerror_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>500</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Meiryo UI&quot;;
color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
